.eslintrc.js

```diff
@@ -1,5 +1,27 @@
-const defaultConfigurations = require("@bigbinary/neeto-commons-frontend/configs/nanos/eslint/index.js");
-const { mergeDeepLeft } = require("ramda");
+const { mergeLeft, mergeDeepLeft } = require("ramda");
+
+const commonConfiguration = require("./configs/eslint");
+
+const nanosConfiguration = {
+  extends: [
+    "plugin:@bigbinary/neeto/nanos-recommended",
```

Comment:
We can also remove the `eslint-plugin-neeto` package because it hasn't been open sourced yet.